plugins/flux/README.md

```diff
@@ -0,0 +1,9 @@
+# FluxCD plugin
+
+This plugin adds completion for [Flux](https://helm.sh/), an open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
```

Comment:
Should we have https://fluxcd.io?

Reply:
I'd also name the plugin `fluxcd`, just to be more semantic in terms of not confusing with other projects named flux, such as https://github.com/influxdata/flux

Reply:
Yes, make sense. I will change it right now